scripts/automation/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
you should also commit the package lock file

Reply:
added